HISTORY.md

```diff
@@ -22,6 +22,7 @@
 ## Bug fixes
 
 * Fixed bug in setting a fill value for var-sized attributes.
+* Fixed error "[TileDB::ChunkedBuffer] Error: Chunk read error; chunk unallocated error" that may occur when writing var-sized attributes.
```

Comment:
Please move this to a 2.0.7 section and add a link to this PR.

Reply:
I had to add a 2.0.7 section with this PR. Let me know if that's not what you intended.

Reply:
Perfect thanks!